schemas/mozilla_nimbus_schemas/experimenter_apis/experiments/feature_manifests.py

```diff
@@ -298,6 +298,15 @@ class SdkFeature(BaseFeature):
         description="The variables that this feature can set."
     )
 
+    allow_coenrollment: bool = Field(
+        alias="allow-coenrollment",
```

Comment:
Why's this one kebab and the other one camel?

Reply:
its already kebab on mobile, but *everything* on desktop is camel /shrug